routes/chat_route.py

```
@@ -22,81 +22,78 @@
-            detail="녹음 파일 전달 실패"
-        )
-    else:
-        user_message = await try_stt(audio_file, redis)
+    user_id = token_user_id          # 토큰 값을 그대로 사용
 
-    print(user_message)
+    # 🎙️ STT 처리
+    try:
+        user_message = await try_stt(audio_file, redis)
+    except Exception as e:
+        raise HTTPException(500, f"STT 변환 실패: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)